src/modules/project/sagas.ts

```diff
@@ -130,42 +132,46 @@ export function* projectSaga(builder: BuilderAPI) {
     }
   }
 
-  function* handleDuplicateProject(action: DuplicateProjectAction) {
+  function* handleDuplicateProjectRequest(action: DuplicateProjectRequestAction) {
     const { project, type } = action.payload
     const ethAddress: string = yield select(getAddress)
     const isTemplatesEnabled: boolean = yield select(getIsTemplatesEnabled)
     const scene: Scene = yield getSceneByProjectId(project.id, type)
 
     let thumbnail: string = project.thumbnail
 
-    if (isTemplatesEnabled && project.isTemplate) {
-      thumbnail = yield call(getImageAsDataUrl, project.thumbnail)
-    } else if (thumbnail && isRemoteURL(thumbnail)) {
-      thumbnail = yield call(getImageAsDataUrl, project.thumbnail)
-    }
+    try {
+      if ((thumbnail && isRemoteURL(thumbnail)) || (isTemplatesEnabled && project.isTemplate)) {
+        thumbnail = yield call(getImageAsDataUrl, project.thumbnail)
```

Comment:
if the project is a `template`, it's possible to load the thumbnail from the public folder and send it to the backend as base64.